src/test/java/network/brightspots/rcv/ContestConfigTests.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * RCTab
+ * Copyright (c) 2017-2023 Bright Spots Developers.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * Purpose: These tests check configuration files.
+ * Design: Unit tests, and other tests that don't run a tabulation..
+ * Conditions: During automated testing.
+ * Version history: see https://github.com/BrightSpots/rcv.
+ */
+
+package network.brightspots.rcv;
+
+import static network.brightspots.rcv.ContestConfigMigration.isVersionNewer;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class ContestConfigTests {
+  @BeforeAll
+  static void setup() {
+    Logger.setup();
+  }
+
+  @Test
+  @DisplayName("test isVersionNewer")
+  void testVersionComparison() {
+    // Basic checks: version numbers work fine
+    assertFalse(isVersionNewer("1.3.0", "1.3.0"));
+    assertFalse(isVersionNewer("1.2.0", "1.3.0"));
+    assertFalse(isVersionNewer("1.3.0", "1.3.1"));
+    assertTrue(isVersionNewer("1.3.1", "1.3.0"));
+
+    // Version comparison ignores the -alpha specifier
+    assertFalse(isVersionNewer("1.2.0-alpha", "1.3.0"));
+    assertTrue(isVersionNewer("1.4.0-alpha", "1.3.0"));
+    assertFalse(isVersionNewer("1.4.0-alpha", "1.4.0"));
+    assertFalse(isVersionNewer("1.4.0", "1.4.0-alpha"));
+
+    // -beta and -alpha are considered the same version
```

Comment:
Please add a comment linking to your issue here as well (thanks for opening that).

I'm undecided on whether or not we should add bespoke code to handle this situation ourselves yet... maybe take a look at how `isVersionNewer` is used in practice and let me know what you think?

Reply:
I'm inclined to leave as-is, since the version isn't really strongly used at all -- at least not yet. Once we implement #706, we might decide the current version is fine? But I don't feel strongly, it won't be hard to support this.